test/e2e/workload/statefulset.go

```diff
@@ -60,7 +60,7 @@ var _ = framework.KubeDescribe("StatefulSet", func() {
 		ns = f.Namespace.Name
 	})
 
-	framework.KubeDescribe("Basic StatefulSet functionality [StatefulSetBasic]", func() {
```

Comment:
This I think is nested within the outer `SIGDescribe` and would add the SIG label twice.